packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -344,9 +343,32 @@ export const Checkout = ({
           })}
         />
 
+        {transactionInProgress && isMobile ? null : (
+          <div className="payment-panel">
+            <PlanDetails
+              {...{
+                selectedPlan,
+                isMobile,
+                showExpandButton: isMobile,
+                coupon,
+              }}
+            />
+
+            <CouponForm
+              {...{
+                planId: selectedPlan.plan_id,
+                readOnly: false,
+                subscriptionInProgress: inProgress || transactionInProgress,
+                coupon,
+                setCoupon,
+              }}
+            />
+          </div>
+        )}
+
         <div
           className={classNames(
-            'checkout-payment component-card border-t-0 pt-4 px-4 pb-14 row-start-2 row-end-3 rounded-t-none text-grey-600',
+            'component-card border-t-0 min-h-full mb-6 pt-4 px-4 pb-14 rounded-t-lg text-grey-600 tablet:rounded-t-none desktop:pt-4 desktop:px-12 desktop:pb-12',
```

Comment:
If we define `pt-4` above, do we also need `desktop:pt-4`?

Reply:
Aah, no we don't. I'll remove the desktop one, thanks @pdehaan!